NineChronicles.Headless.Executable/Commands/TxCommand.cs

```diff
@@ -65,10 +70,11 @@ public TxCommand(IConsole console)
                     nameof(MonsterCollect) => new MonsterCollect(),
                     nameof(ClaimMonsterCollectionReward) => new ClaimMonsterCollectionReward(),
                     nameof(Stake) => new Stake(),
-                    // FIXME: This `ClaimStakeReward` cases need to reduce to one case.
-                    nameof(ClaimStakeReward1) => new ClaimStakeReward1(),
-                    nameof(ClaimStakeReward) => new ClaimStakeReward(),
-                    nameof(ClaimStakeReward3) => new ClaimStakeReward3(),
+                    // FIXME: How to set default action version? Can I use blockIndex instead of action version?
+                    "ClaimStakeReward" => (ActionBase)ClaimStakeRewardFactory.CreateByVersion(
+                        actionVersion ?? 2,
+                        new Address()
+                    ),
```

Comment:
I think a default version is not exist. So.. how about this?

```suggestion
                    "ClaimStakeReward" => actionVersion
                        ? (ActionBase)ClaimStakeRewardFactory.CreateByVersion(
                            actionVersion,
                            new Address())
                        : new ClaimStakeReward(new Addess()),
```

There is a comment in `ClaimStakeRewardFactory.CreateByBlockIndex()` like below. And it make some confuse. I'll fix this!

```cs
// FIXME: This method should consider the starting block index of
//        `claim_stake_reward2`. And if the `blockIndex` is less than
//        the starting block index, it should throw an exception.
// default: Version 2
return new ClaimStakeReward(avatarAddress);
```